cs/nuget/scripts/10-install-stack.ps1

```diff
@@ -0,0 +1,24 @@
+[CmdletBinding()]
```

Comment:
Is this script just used in the internal-to-Microsoft VSTS build for official releases?

Reply:
Yes, it is.